CLAC

```diff
@@ -0,0 +1,157 @@
+                       DEDIS / EPFL / Lausanne / CH
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+                      http://www.apache.org/licenses/
+
+   Thank you for your interest in software from DEDIS (Lab).
```

Comment:
Instead of speaking of "the Lab" everywhere wouldn't it be more precise when replacing it by "DEDIS" or by "the DEDIS Lab"?